### PR TITLE
feat: add document.get_elements_by_tag_name

### PR DESCRIPTION
**Pull request by dinkelspiel**

### CODE REVIEW

**Comment by CrowdHailer** [comment]
Set timeout already exists on global.

File: src/plinth/browser/document.gleam [hunk 1]
```diff
@@ -23,6 +23,9 @@ pub fn body() -> Element
 @external(javascript, "../../document_ffi.mjs", "getElementById")
 pub fn get_element_by_id(id: String) -> Result(Element, Nil)
 
+@external(javascript, "../../document_ffi.mjs", "getElementsByTagName")
+pub fn get_elements_by_tag_name(tag_name: String) -> Result(List(Element), Nil)
```
Comment (CrowdHailer) [inline src/plinth/browser/document.gleam:27]: Can this return an Array, like `query_selector_all` I prefer doing minimal work in plinth and we already rely on gleam_javascript to array.to_list is already available in Gleam land

Comment (dinkelspiel) [reply src/plinth/browser/document.gleam]: Fixed

**Comment by CrowdHailer** [comment]
looks good thank you

**Pull request merged**